src/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import './LandingPage.css';
+import axios from "axios";
+import { setUserLoggedIn } from "../actions";
+import { connect } from "react-redux";
+import RegisterModal from '../LoginPage/components/RegisterModal';
+
+
+class LandingPage extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            notVerified: false
+        };
+    };
+
+    goToFeedPage() {
+        this.props.history.push("/feed");
+    };
+
+
+    login(e) {
+        //Stops page from refreshing when the form is submitted.
+        e.preventDefault();
+        const email = document.getElementById("email").value;
+        const password = document.getElementById("password").value;
+        // this.props.createSession(email, password);
+        axios
+            .post(process.env.REACT_APP_API_ENDPOINT + "sessions", { email, password })
+            .then(response => {
+                sessionStorage.setItem("confirmed", response.data.confirmed_at);
+                sessionStorage.setItem("userId", response.data.id);
+                // sessionStorage.setItem('isAdmin', response.data.admin);
+                sessionStorage.setItem("adminUser", response.data.admin);
+
+                this.props.setUserLoggedIn(true, response.data.id);
+
+                response.data.confirmed_at
```

Comment:
If you want to do this as a ternary expression, can you collapse this to a single line? Alternatively, let's make it a more explicit if-else block.

Reply:
Yeah, I'd use an explicit if-else in this case.

Reply:
done